ui/src/cloud/utils/limits.ts

```diff
@@ -34,40 +34,33 @@ export const extractTaskMax = (limits: LimitsState): number => {
   return get(limits, 'tasks.maxAllowed') || Infinity
 }
 
+export const extractChecksLimits = (limits: LimitsState): LimitStatus => {
+  return get(limits, 'checks.limitStatus')
+}
 export const extractChecksMax = (limits: LimitsState): number => {
   return get(limits, 'checks.maxAllowed') || Infinity
 }
 
+export const extractRulesLimits = (limits: LimitsState): LimitStatus => {
+  return get(limits, 'rules.limitStatus')
+}
 export const extractRulesMax = (limits: LimitsState): number => {
   return get(limits, 'rules.maxAllowed') || Infinity
 }
 export const extractBlockedRules = (limits: LimitsState): string[] => {
   return get(limits, 'rules.blocked') || []
 }
 
+export const extractEndpointsLimits = (limits: LimitsState): LimitStatus => {
+  return get(limits, 'endpoints.limitStatus')
```

Comment:
should these return some defaults?

Reply:
I don't think so. The defaults are set to `LimitStatus.OK` so there's always a value when these gets are called.